scripts/build_osx_universal_gmpfr.sh

```diff
@@ -5,15 +5,16 @@ set -e
 # Assumes that the user has enough rights to run brew fetch
 
 # Downloading
+SED_PGM='s/^Downloaded to: |^Already downloaded: //p'
```

Comment:
This is because we are doing in `gudhi-devel/.github/workflows/pip-build-osx.yml`:
```
          brew install boost eigen gmp mpfr cgal || true
     ...
          ./scripts/build_osx_universal_gmpfr.sh
```
I am ok with the regular expression